Sources/BraintreePayPal/BTPayPalReturnURL.swift

```diff
@@ -39,7 +39,9 @@ struct BTPayPalReturnURL {
     /// - Parameter url: an app switch or ASWebAuthenticationSession return URL
     /// - Returns: `true` if the url represents a valid PayPal app switch return
     static func isValid(_ url: URL) -> Bool {
-        url.scheme == "https" && (url.path.contains("cancel") || url.path.contains("success"))
+        url.scheme == "https"
+        && (url.path.contains("cancel") || url.path.contains("success"))
```

Comment:
curious if there's a reason we don't check for `url.path.contains("error")` here

Reply:
That's a great question. Currently PayPal never sends us an error path, only cancel or success.